Donation-platform/src/main/java/com/google/charities/AddCharitiesFromJSON.java

```diff
@@ -0,0 +1,165 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+ 
+package com.google.charities;
+ 
+import com.google.DbCalls;
+import com.google.charities.CharityClone;
+import com.google.gson.Gson;
+import java.io.FileReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.Reader;
+import java.util.Arrays;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.model.Charity;
+import com.google.model.Tag;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.ArrayList;
+import java.util.List;
+
+
+
+public final class AddCharitiesFromJSON {
+ 
+  private DatastoreService ds;
+  private DbCalls db;
+  
+  // TAG NAMES
+  private static final String ANIMAL = "Animals";
+  private static final String ARTS = "Arts, Culture, Humanities";
+  private static final String EDU = "Education";
+  private static final String ENV = "Environment";
+  private static final String HEALTH = "Health";
+  private static final String HUMAN = "Human Services";
+  private static final String INTL = "International";
+  private static final String RIGHTS = "Human and Civil Rights";
+  private static final String RELI = "Religion";
+  private static final String COMM = "Community Development";
+  private static final String RESEARCH = "Research and Public Policy";
+ 
+  // TAG IMAGES
```

Comment:
we should store these images in res/ folder. Its possible tomorrow the content of these links might change

Reply:
We also need to consolidate the images we want to use.

Reply:
We have since consolidated the images we're using. I have tried storing / accessing the images in our resources folder, but the problem is that that we store images as a string in the Charity class. This means that we find the absolute path in AddCharitiesFromJSON, but this path does not work (gives file not found error) when the browse page tries to access it. I can continue to try to resolve this but given the limited time left I feel it might be better to just continue using the links we have.